crates/circuit/src/circuit_data.rs

```diff
@@ -1626,3 +1975,133 @@ impl<'py> FromPyObject<'py> for AssignParam {
         Ok(Self(Param::extract_no_coerce(ob)?))
     }
 }
+
+/// Get the list of bits referred to by the specifier ``specifier``.
+///
+/// Valid types for ``specifier`` are integers, bits of the correct type (as given in ``type_``), or
+/// iterables of one of those two scalar types.  Integers are interpreted as indices into the
+/// sequence ``bit_sequence``.  All allowed bits must be in ``bit_set`` (which should implement
+/// fast lookup), which is assumed to contain the same bits as ``bit_sequence``.
+///
+/// Returns:
+///     List[Bit]: a list of the specified bits from ``bits``.
```

Comment:
I think this is a Rust-only function, right? But the doc string is written in Python style.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/13860/commits/478cd0171c337e4d69cfd44edbdacb2dd3f7deb2